content/blog/architect-aws-application-infra-with-pulumi-stack-references/index.md

```diff
@@ -0,0 +1,457 @@
+---
+title: "Architect AWS Application Infrastructure with Pulumi Stack References"
+date: "2019-06-14"
+meta_desc: "How to architect your AWS infrastructure to optimize team collaboration with Pulumi Stack References"
+meta_image: "dashboard.png"
+authors: ["paul-stack"]
+tags: ["AWS","stack-reference", "architecture"]
+---
+
+In this post, we will talk about the best way to architect your Pulumi applications. We are going to build out the following
+infrastructure in AWS:
+
+* AWS Fargate service that does not serve traffic directly
+* AWS ALB as the entry point to the Fargate Service
+* AWS RDS Instance that is stored in a separate network from the Application and does not service traffic directly from the internet
+
+To do this, we are going to split the infrastructure into 2 AWS VPCs. 1 VPC will be used for the application based resources 
+(e.g Fargate Service, ALB) and the other VPC will be for the database resources. The VPCs will be peered to allow communication
+between them so that the application can communicate with the databse securely.
+
+![application-architecture](application-architecture.png)
+
+## Layer it up
+
+When we think of infrastructure, we tend to think of layers. A layer is a logical set of components / resources that can be
```

Comment:
Maybe something here to the effect that layers often break down to different team responsibilities in larger orgs?